console/app_serve.go

```diff
@@ -60,5 +62,19 @@ func (s AppServe) getPortAddr(app inter.App) string {
 	} else {
 		port = app.Make("config.App.Port").(int)
 	}
-	return ":" + strconv.Itoa(port)
+	return strconv.Itoa(port)
+}
+
+func (s AppServe) getHostAddr(app inter.App) string {
+	var host string
+	if len(s.Host) != 0 {
+		host = s.Host
+	} else {
+		host = "127.0.0.1"
```

Comment:
```suggestion
		host = ""
```
In the old situation it listens to every host, but now you limit it to 1 host right? Perhaps it is better not to use `127.0.0.1:port` by default, but only `:port` (see suggestion).

It is also good for me to know your use case.